catalog/resource_quality_monitor.go

```diff
@@ -53,6 +53,7 @@ func ResourceQualityMonitor() common.Resource {
 			common.CustomizeSchemaPath(m, "profile_metrics_table_name").SetReadOnly()
 			common.CustomizeSchemaPath(m, "status").SetReadOnly()
 			common.CustomizeSchemaPath(m, "dashboard_id").SetReadOnly()
+			common.CustomizeSchemaPath(m, "schedule", "pause_status").SetReadOnly()
```

Comment:
I don't see a related designation for the overall `schedule` resource - is it set as an optional field overall and this only marks the subfield as read only?

Reply:
yes, `schedule` is optional automatically if it's marked as `omitempty`. This one is marked as read-only because it comes from backend, and isn't set by user